packages/svelte/src/internal/server/index.js

```diff
@@ -64,6 +64,8 @@ export const VoidElements = new Set([
 	'wbr'
 ]);
 
+export const RawTextElements = new Set(['textarea', 'script', 'style', 'title']);
```

Comment:
Please can you put these in `constants.js` where all the others are? Plus it doesn't need to be a Set, an array here would be slightly faster than a Set as there are so few elements in it.

Reply:
Thanks, I have moved and changed it now. I had just used the same method as the `VoidElements` set that was above.